### PR TITLE
fix(common): update CLDR generated files

### PR DESCRIPTION
**Pull request by josephperrott**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -843,7 +843,7 @@ workflows:
             - setup
       - test_aio_preview:
           requires:
-            - aio_preview
+           - aio_preview
```
Comment (gkalpak) [inline .circleci/config.yml]: 😱

File: tools/gulp-tasks/cldr/extract.js [hunk 2]
```diff
@@ -519,7 +519,7 @@ function getPluralFunction(locale) {
 
   fn = fn
     .replace(
-      toRegExp('function anonymous(n\n/*``*/) {\n'),
+      toRegExp('function anonymous\(n\n\) {\n'),
```
Comment (gkalpak) [inline tools/gulp-tasks/cldr/extract.js]: Escaping `(` and `)` is handled by `isRegExp()`, so it shouldn't be necessary.

**Comment by clydin** [comment]
Can you add the CLDR version into the commit message?  The large volume of changes makes it hard to find what version is being used.

**Review by petebacondarwin — APPROVED** [review]
@josephperrott - can the first commit message be

```
build: update CLDR version to v33
```

Also can you explain what you did in the final fixup commit that changes `\u202f` strings, and why?

**Comment by josephperrott** [comment]
Updated the commit messages to include the CLDR version.

@petebacondarwin I don't know what I did that ended up removing them, but that was the wrong thing since those strings should be there.

**Review by petebacondarwin — APPROVED** [review]

**Comment by doom777** [comment]
forgot to remove the old cldr-data directory from .gitignore.
I'll remove it in my PR

**Pull request closed without merging**